doc/opcode.md

```diff
@@ -70,6 +70,7 @@ with `"`, either `OP_EXT1` or `OP_EXT2` or `OP_EXT2` can be prefixed.
 | OP_JMPIF'        | BS           | if R(a) pc=b                                           |
 | OP_JMPNOT'       | BS           | if !R(a) pc=b                                          |
 | OP_JMPNIL'       | BS           | if R(a)==nil pc=b                                      |
+| OP_JUW           | S            | unwind_and_jump_to(a)                                  |
```

Comment:
I don't like the name `OP_JUW`. It perhaps stands for "Jump While Unwind", but it's not a common acronym and is hard to remember. Maybe we should name it with a better (and more descriptive) term. At least it should contain "JMP" if it's a (kind of) jump.

Reply:
Certainly it is hard to understand.
`OP_JMPUNWIND` is clear, but I think it's a little long.
That said, I can't think of anything else, and I have an instruction that is as long as that, so I'll change it.

Reply:
How about 'OP_JMPUW`. It's a bit hard to guess 'UW' stands for but better than `JUW` since we can tell it's a jump.

Reply:
@dearblue If it's OK for you. I will update since I am merging this PR to the `mruby3` branch right now.

Reply:
Merging to the `mruby3` branch is fine.
Please let me know if you have any necessary work.

Reply:
No problem. I am working on it. I also made corrections `_hander` to `_handler`. Is it OK?

Reply:
That is my mistake. Please replace it.